tests/micro/test_runtime_micro_on_arm.py

```diff
@@ -33,7 +33,7 @@
 # Ex : export CMSIS_ST_PATH="/home/yourid/st/STM32Cube_FW_F7_V1.16.0/Drivers/CMSIS"
 DEV_CONFIG_A = micro.device.arm.stm32f746xx.generate_config("127.0.0.1", 6666)
 DEV_CONFIG_B = micro.device.arm.stm32f746xx.generate_config("127.0.0.1", 6666)
-TARGET = 'c -device=micro_dev'
+TARGET = 'micro_dev'
```

Comment:
breaking change: now `micro_dev` is a new target, instead of the `device_name` of target `c`

Reply:
I guess the other option would be to add a --runtime attr to c codegen, but I can also do that in a follow-up

Reply:
Sure thing, let's discuss about it in the future :-)